daemons/systemd/armadietto.service

```diff
@@ -0,0 +1,22 @@
+[Unit]
+Description=Armadietto RemoteStorage server
+Requires=network.target
+After=network.target
+StartLimitIntervalSec=0
+Documentation=https://github.com/remotestorage/armadietto/
+
+[Service]
+Type=simple
+Restart=always
+RestartSec=1
+User=armadietto
+Group=armadietto
+Environment=
+ExecStartPre=
```

Comment:
Is there a reason for including empty config variables?

Reply:
If there is a need to add environment variables (such as DEBUG=true) or a better shutdown command, you don't have to look up the variable name.

We can omit them if you feel strongly about it.

Reply:
Hmm, I can see how that could be useful. Except that one still has to look up how to actually specify the environment variables themselves in this file, if they're not familiar with systemd unit syntax.

I don't feel strongly about it at all, especially since it's only an example file anyway. Was just interested in the reasoning.